flagengine/integration_test/engine_test.go

```diff
@@ -1,3 +1,7 @@
+// cgo.go
```

Comment:
What are these commented out lines for?

Reply:
Well, they were there to prevent go build hook errors, but digging into it a bit more,  it's clear that it's not very idiomatic to have integration tests in a different directory/package, hence I decided to move it a level up.